embassy-nrf/src/pwm.rs

```diff
@@ -81,31 +335,43 @@ impl<'d, T: Instance> Pwm<'d, T> {
             .ptr
             .write(|w| unsafe { w.bits(&s.duty as *const _ as u32) });
         r.seq0.cnt.write(|w| unsafe { w.bits(4) });
-        r.seq0.refresh.write(|w| unsafe { w.bits(32) });
+        r.seq0.refresh.write(|w| unsafe { w.bits(0) });
         r.seq0.enddelay.write(|w| unsafe { w.bits(0) });
 
         r.decoder.write(|w| {
             w.load().individual();
             w.mode().refresh_count()
         });
         r.mode.write(|w| w.updown().up());
-        r.prescaler.write(|w| w.prescaler().div_1());
-        r.countertop
-            .write(|w| unsafe { w.countertop().bits(32767) });
+        r.prescaler.write(|w| w.prescaler().div_16());
```

Comment:
why these changes?

Reply:
Doesnt so much matter, as long as theyre documented, I was just looking at what nrf sdk uses as its 'defaults' and used them

Reply:
Another option is to have a default for SimplePwm as well, that impls default with some values. Though still pushes the question what should the default values be, the nrf sdk or just what we had here before

Reply:
FWIW re. Saadc, I put Embassy's defaults back to what the nRF sdk does.

Reply:
Ah, well then ill leave this with nrf sdk's defaults as well then and resolve this.